src/entries/popup/pages/home/Activity/ActivityValue.tsx

```diff
@@ -87,17 +99,23 @@ const activityValues = (transaction: RainbowTransaction) => {
   const { balance, native } = asset;
   if (balance.amount === '0') return;
 
-  const formatOptions =
-    +balance.amount > 100_000 ? ({ notation: 'compact' } as const) : undefined;
-  const assetValue = `${formatNumber(balance.amount, formatOptions)} ${
-    asset.symbol
-  }`;
+  const assetValue = isSuperTinyValue(balance.amount)
+    ? `>${formatNumber(SUPER_TINY_THRESHOLD)} ${asset.symbol}`
```

Comment:
Maybe im confused but shouldnt this be '<' (less than) rather than '>' (greater than)?

```suggestion
    ? `<${formatNumber(SUPER_TINY_THRESHOLD)} ${asset.symbol}`
```